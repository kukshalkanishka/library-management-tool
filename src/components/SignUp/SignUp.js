import React from "react";

const SIGN_UP_TITLE = "SIGN UP";

const SignUp = () => {
  return (
    <form method="POST" action="/signup" className="form">
      <h3>{SIGN_UP_TITLE}</h3>
      <input type="text" placeholder="username" name="username" required />
      <input type="password" placeholder="password" name="password" required />
      <input type="submit" value="SIGN UP"/>
    </form>
  );
};

export default SignUp;
