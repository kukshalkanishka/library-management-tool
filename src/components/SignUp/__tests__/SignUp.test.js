import React from "react";
import { shallow, configure } from "enzyme";
import SignUp from "../SignUp";

import Adapter from "enzyme-adapter-react-16";

configure({ adapter: new Adapter() });

describe("SignUp", () => {
  it("should render properly", () => {
    const wrapper = shallow(<SignUp />);
    expect(wrapper.debug()).toMatchSnapshot();
  });
});
