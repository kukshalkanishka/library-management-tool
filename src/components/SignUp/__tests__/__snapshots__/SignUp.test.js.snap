// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignUp should render properly 1`] = `
"<form method=\\"POST\\" action=\\"/signup\\" className=\\"form\\">
  <h3>
    SIGN UP
  </h3>
  <input type=\\"text\\" placeholder=\\"username\\" name=\\"username\\" required={true} />
  <input type=\\"password\\" placeholder=\\"password\\" name=\\"password\\" required={true} />
  <input type=\\"submit\\" value=\\"SIGN UP\\" />
</form>"
`;
